pages/docs/getting-started/beta/quick-installation.md

```diff
@@ -10,40 +10,101 @@ Installing Open Data Hub requires OpenShift Container Platform version 4.8+. All
+
+From ODH 1.4.1 on, we provide a mix-in configmap trusted-cabundle-odh with the OLM package that via the Cluster Network Operator merges the user-provided (additionally-trusted) and system CA certificates coming from the operating system of the cluster nodes into a single CA-bundle file in the configmap trusted-ca-bundle-odh that we reference in the [operator subscription spec.config](https://docs.openshift.com/container-platform/4.10/operators/admin/olm-configuring-proxy-support.html#olm-inject-custom-ca_olm-configuring-proxy-support).
+
+This is especially useful if you want to download manifest.tar.gz files from an enterprise-internal server location with private PKI-based SSL that is not publicly-trusted. Also, the publicly-trusted CAs as they are mixed in from the cluster network operator tend to be more up-to-date than what is contained within docker images.
+
+```
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: opendatahub-operator
+  namespace: openshift-operators
+spec:
+  channel: stable
+  installPlanApproval: Automatic
+  name: opendatahub-operator
+  source: community-operators
+  sourceNamespace: openshift-marketplace
+  startingCSV: opendatahub-operator.v1.4.1
+  config: 
+      selector:
+        matchLabels:
+          name: opendatahub-operator
+      volumes: 
+      - name: trusted-cabundle
+        configMap:
+          name: trusted-cabundle-odh
+          items:
+            - key: ca-bundle.crt 
+              path: tls-ca-bundle.pem
+          optional: true
+      volumeMounts: 
+      - name: trusted-cabundle
+        mountPath: /etc/pki/ca-trust/extracted/pem
+        readOnly: true
+```
+
+
+
+
 ### Create a New Open Data Hub Deployment
 
 The Open Data Hub operator will create new Open Data Hub deployments and manage its components.  Let's create a new Open Data Hub deployment.
 
-1. Find the Open Data Hub Operator under `Installed Operators` (inside the namespace you created earlier)
+1. Find the Open Data Hub Operator under `Installed Operators` (inside the namespace you created earlier).
+![Change Namespace to 'odh']({{site.baseurl}}/assets/img/pages/docs/quick-installation/odh-namespace.png "Change Namespace to 'odh'")
 ![Installed Operators]({{site.baseurl}}/assets/img/pages/docs/quick-installation/installed-operators.png "Installed Operators")
 
 1. Click on the Open Data Hub Operator to bring up the details for the version that is currently installed.
 ![Open Data Hub Operator]({{site.baseurl}}/assets/img/pages/docs/quick-installation/odh-operator.png "Open Data Hub Operator")
 
-1. Click `Create Instance` to create a new deployment.
+1. Click `Create Instance` to create a new deployment. **Double-check that you really switched to your newly-created namespace / project and that openshift-operators is not the current project.**
```

Comment:
The warning about the duplicate `operatorgroup` if the target namespace is `openshift-operators` should be here

Reply:
Agreed, i.e. that "Installing in the "OpenShift Operators" namespace will result in a fatal error similar to "csv created in namespace with multiple operatorgroups, can't pick one automatically".